src/core/Warp.ts

```diff
@@ -67,8 +58,11 @@ export class Warp {
   ) {
     this.createContract = new DefaultCreateContract(arweave, this);
     this.testing = new Testing(arweave);
-    this.kvStorageFactory = (contractTxId: string) => {
-      return new TrieLevel(new Level(`${DEFAULT_LEVEL_DB_LOCATION}/kv/ldb/${contractTxId}`));
+    this.kvStorageFactory = (key: string) => {
+      return new LevelDbCache({
+        inMemory: false,
```

Comment:
set to true for warp.environement = 'local'

..and then remove manual cache deletion from the integration test.

Reply:
There are some side effects in this approach, some of the tests are not taking this change very well. Let's skip it for now.